src/main/java/com/twitter/whiskey/util/ZlibInflater.java

```diff
@@ -171,13 +171,14 @@ public void reset() {
 
     @Override
     public void end() {
-        inflater.end();
+        if (inflater != null) inflater.end();
+        super.end();
```

Comment:
Is this "super.end()" call necessary? I'm not sure why this class both has an Inflator and extends Inflator. I don't see any other super calls (besides finalize), so it seems like no functionality from super is used.